src/Error/ErrorLogger.php

```diff
@@ -58,7 +58,7 @@ public function __construct(array $config = [])
     /**
      * @inheritDoc
      */
-    public function log($level, Stringable|string $message, array $context = []): void
+    public function log(string|int $level, Stringable|string $message, array $context = []): void
```

Comment:
We are using the trait from the PSR package which doesn't have these type declarations hence we need to follow the same.